src/contexts/FirstContext.js

```diff
@@ -0,0 +1,29 @@
+import { createContext, useState } from 'react';
+
+export const FirstContext = createContext();
+
+const FirstContextProvider = (props) => {
+
+  const [weatherValues, setWeatherValue] = useState([
+    {name: 'Monday', Precipitation: 20, WindSpeed: 65},
```

Comment:
Can we update these strings to be dates? https://c3js.org/samples/axes_x_localtime.html